src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -2,23 +2,161 @@ import React from "react"
 import "react-native"
 import * as renderer from "react-test-renderer"
 
+jest.mock("../../../../metaphysics", () => ({ metaphysics: jest.fn() }))
+import { metaphysics } from "../../../../metaphysics"
+const mockphysics = metaphysics as jest.Mock<any>
+
 import { ConfirmBid } from "../ConfirmBid"
+jest.unmock("react-relay")
+import relay from "react-relay"
+
+const saleArtwork = {
+  artwork: {
+    id: "meteor shower",
+    title: "Meteor Shower",
+    date: "2015",
+    artist_names: "Makiko Kudo",
+  },
+  sale: {
+    id: "best art sale in town",
```

Comment:
#minor since I know this is a test, but since it's an id this would be returned in the form `"best-art-sale-in-town"`

Reply:
Cool, will do.